scripts/migrate-vfolder-structure.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from pathlib import Path
+
+
+def main():
+    input_path = Path(sys.argv[1])  # full path for vroot/local
+    output_path = Path(sys.argv[2])  # full path for volume directory. ex. vfs/
+    print(input_path, output_path)
+    subfolders = [x for x in input_path.iterdir() if x.is_dir()]
+
+    print("Subfolders ", subfolders)
+
+    for folder in subfolders:
+        folder = str(folder)
+        folder = folder.split("/")[-1]
+        header_dir = Path(folder[0:2])
+        second_lvl_dir = Path(folder[2:4])
+        rest_dir = Path(folder[4:])
+
+        root = Path(input_path / Path(folder))
+        list_of_files = [x for x in root.iterdir() if x.is_file()]
+        print("list of files ", list_of_files)
```

Comment:
I think this is also unnecessary.
There may be vfolders with many number of files, which would drastically slow down this operation and fill up the user's terminal with those file names.